docs/initialization.md

```diff
@@ -50,10 +50,10 @@ ebcli tx ethbridge create-claim --help
 # Now you can test out the ethbridge module by submitting a claim for an ethereum prophecy
 # Create a bridge lock claim (Ethereum prophecies are stored on the blockchain with an identifier created by concatenating the nonce and sender address)
 # ebcli tx ethbridge create-claim [ethereum-chain-id] [bridge-contract] [nonce] [symbol] [token-contract] [ethereum-sender-address] [cosmos-receiver-address] [validator-address] [amount]",
```

Comment:
should update the new `--help` output too

Reply:
https://github.com/cosmos/peggy/pull/175/commits/a1bed706b60a52d3c88b8e3be332e1c0a1979959, https://github.com/cosmos/peggy/pull/175/commits/afdffa5a7bd17853d6f81f38b486e610ab3f36a2

Reply:
looks like both updates should use `bridge-registry-contract` instead

Reply:
initialization.md has been deprecated in favor of setup-bridge-chain.md. `bridge-registry-contract` revisions to setup-bridge-chain.md are in https://github.com/cosmos/peggy/pull/175/commits/9879683359d2d8abe7b89825447ba7d2906470bb